nntrainer/tensor/tensor.cpp

```diff
@@ -746,12 +756,22 @@ Tensor &Tensor::sum(const std::vector<unsigned int> &axes, Tensor &output,
   if (axes.size() == 1) {
     this->sum(axes[0], output, alpha);
   } else {
-    Tensor ret = this->sum(axes[0], alpha);
+    /** club axes together */
+    Tensor new_reshaped = *this;
+    std::vector<unsigned int> new_axes = {axes[0]};
+    for (unsigned int i = 1; i < axes.size(); ++i) {
+      if (axes[i] == axes[i - 1] + 1) {
```

Comment:
note) it is not guaranteed that axes are always in sorted order. This is just an optimization so I think it's fine though

Reply:
Yes, its not gauranteed. Developer must try to give it in order for optimization. We can add sort and unique but IMO its overkill.